plugins/tiddlywiki/dynannotate/examples/simple.tid

```diff
@@ -10,75 +10,68 @@ caption: Simple
 $example$
 \end
 
-<div class="tc-dynannotation-example-info">
-
-!! Simple annotation
-
-We use the `target*` attributes to specify a target string for the annotation and optionally a prefix and suffix for disambiguating multiple occurances.
-
-</div>
-
-<<show-example """
+\define search-example-code()
+\rules only filteredtranscludeinline
```

Comment:
Wait, what? Did you just suggest using macro substitutions? 😱 
Just kidding. But do we really need to have _more_ of those around?
I'd like to just add a short explanation to the code instead, something like
```
<!-- The rules pragma is used here to evaluate only the inline filters depending on the chosen configuration, but leave everything else untouched, to be displayed within the example code block.
This way the displayed code updates to reflect the choices the user has made in the example UI. -->
```

Reply:
I'd be happy with the addition of a comment. I don't think we should compromise the readability of the example output to make the wikitext easier to read, but otherwise anything you can do is appreciated.